src/main/java/interface_adaptors/RecordViewModel.java

```diff
@@ -1,4 +1,99 @@
 package interface_adaptors;
 
-public class RecordViewModel {
+import javax.imageio.ImageIO;
+import javax.swing.*;
+import java.awt.*;
+import java.util.ArrayList;
+
+import entities.Record;
```

Comment:
entities shouldn't be in imported/ manipulated with in the interface_adapters layer